ephios/plugins/basesignup/signup/common.py

```diff
@@ -81,7 +81,9 @@ def _participant_checkers(self):
     @staticmethod
     def check_qualification(method, participant):
         if not participant.has_qualifications(method.configuration.required_qualifications):
-            return ParticipantUnfitError(_("You are not qualified."))
+            return ParticipantUnfitError(
+                _("You don't have all required qualifications for this shift.")
```

Comment:
Again, I don't think the plural is nice. I like the differentiation for event type vs shift in the phrasing. Something like
`You lack the necessary qualification to participate in this shift.` might be better suited.